tutorials/solution_strategies.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<p>Would it work to pass two different parameter objects, one for the model and one for numerics? I know we don't have good practices for this, but perhaps now is a good time to get started.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I'd love to do that if I were confident about which parameters to put where. Shall we make an issue?</p>

Reply:
Good idea.

Reply:
#1210